lib/string-checker.js

```diff
@@ -76,6 +76,41 @@ StringChecker.prototype = {
     },
 
     /**
+     * Parses source code into an AST
+     *
+     * @param  {String} str the source code to parse
+     * @return {Object}     the output AST
+     * @throws Error on invalid source code.
+     */
+    _parse: function(str) {
+        var hashbang = (str[0] === '#' && str[1] === '!');
+        var tree;
+
+        // Convert bin annotation to a comment
+        if (hashbang) {
+            str = '//' + str.substr(2, str.length);
```

Comment:
because then you're creating multiple line breaks which throws off `disallowMultipleLineBreaks`